src/style.css

```
@@ -1 +1,93 @@
-/* add styles here */
+@font-face {
+    font-family: Avenir-Book;
+    src: url(./fonts/Avenir-Book.ttf);
+    font-weight: 300;
+    font-style: normal;
+  }
+  
+  @font-face {
+    font-family: Avenir-Heavy;
+    src: url(./fonts/Avenir-Heavy.ttf);
+    font-weight: 900;
+    font-style: normal;
+  }
+  
+  * {
+    box-sizing: border-box;
+  }
+  
+  html {
+    font-family: Avenir-Book, Avenir-Heavy, sans-serif;
+  }
+  
+  .form_base {
+    margin-bottom: 20px;
+    margin-top: 20px;
+    position: relative;
+  }
+  
+  .input_base {
+    outline: none;
+  }
```

Need to fix it everywhere
```suggestion
.input_base {
  outline: none;
}
```